src/interactive/Tabs.jsx

```diff
@@ -7,13 +7,21 @@ import cx from 'classnames';
  */
 export class TabsTab extends React.Component {
 	render() {
-		const { children, isSelected, className, ...other } = this.props;
+		const {
+			children,
+			isSelected,
+			className,
+			isLeftSelectorLayout,
```

Comment:
can you simplify this name to `isLeft` :)

Reply:
ok I just was afraid it won't be obvious what is left